tests/roles/backend_services/templates/openstack_control_plane.j2

```diff
@@ -148,6 +157,29 @@ spec:
                 metallb.universe.tf/loadBalancerIPs: {{ internalapi_prefix | default('172.17.0') }}.86
             spec:
               type: LoadBalancer
+      # TODO(bogdando): iterate based on renamed_cells value in kustomization.yaml
+      rabbitmq-cell2:
+        persistence:
+          storage: 1Gi
+        override:
+          service:
+            metadata:
+              annotations:
+                metallb.universe.tf/address-pool: internalapi
+                metallb.universe.tf/loadBalancerIPs: 172.17.0.87
+            spec:
+              type: LoadBalancer
+      rabbitmq-cell3:
+        persistence:
+          storage: 1Gi
+        override:
+          service:
+            metadata:
+              annotations:
+                metallb.universe.tf/address-pool: internalapi
+                metallb.universe.tf/loadBalancerIPs: 172.17.0.88
```

Comment:
```suggestion
                metallb.universe.tf/loadBalancerIPs: {{ internalapi_prefix | default('172.17.0') }}.88
```

Reply:
I will address in a follow up, I didn't plan to reiterate with CI as it is quite a heavy run

Reply:
oh, OK, already got a merge confict. So I will fix this and push an update